apps/backend/src/websocket/handlers/judging.ts

```diff
@@ -124,8 +124,11 @@ export const handleUpdateSessionTeam = async (
   await db.updateSession({ _id: session._id }, { teamId: teamId ? new ObjectId(teamId) : null });
 
   callback({ ok: true });
+  const oldSession = { ...session };
   session = await db.getSession({ _id: new ObjectId(sessionId) });
   namespace.to('judging').emit('judgingSessionUpdated', session);
+  if (session.scheduledTime !== oldSession.scheduledTime)
```

Comment:
This event updated the session team. No if statement needed - always emit that the team was updated.